tpdatasrc/tpgamefiles/scr/tpModifiers/draconic_heritage.py

```diff
@@ -0,0 +1,78 @@
+from templeplus.pymod import PythonModifier
+from toee import *
+import tpdp
+import char_class_utils
+import heritage_feat_utils
+
+# Draconic Heritage: Complete Arcane, p. 77
+
+print "Registering Draconic Heritage"
+
+def getDraconicHeritageClassSkill(heritage):
+    skillDict = {
+    heritage_draconic_black: skill_hide,
+    heritage_draconic_blue: skill_listen,
+    heritage_draconic_green: skill_move_silently,
+    heritage_draconic_red: skill_intimidate,
+    heritage_draconic_white: skill_balance,
+    heritage_draconic_brass: skill_gather_information,
+    heritage_draconic_bronze: skill_wilderness_lore,
+    heritage_draconic_copper: skill_hide,
+    heritage_draconic_gold: skill_heal,
+    heritage_draconic_silver: skill_disguise
+    }
+    return skillDict[heritage]
+
+def getSaveDescriptor(heritageElement):
+    if heritageElement == D20DT_ACID:
+        return D20STD_F_SPELL_DESCRIPTOR_ACID
+    elif heritageElement == D20DT_COLD:
+        return D20STD_F_SPELL_DESCRIPTOR_COLD
+    elif heritageElement == D20DT_ELECTRICITY:
+        return D20STD_F_SPELL_DESCRIPTOR_ELECTRICITY
+    elif heritageElement == D20DT_FIRE:
+        return D20STD_F_SPELL_DESCRIPTOR_FIRE
+    return 0
+
+def addClassSkill(attachee, args, evt_obj):
+    heritage = args.get_arg(1)
+    skillEnum = getDraconicHeritageClassSkill(heritage)
+    if evt_obj.data1 == skillEnum:
+        evt_obj.return_val = 1
+    return 0
+
+def addSavingThrowBonus(attachee, args, evt_obj):
+    if attachee.stat_level_get(stat_level_sorcerer) > 0:
+        heritage = args.get_arg(1)
+        heritageElement = heritage_feat_utils.getDraconicHeritageElement(heritage)
+        saveDescriptor = getSaveDescriptor(heritageElement)
+        flags = evt_obj.flags
+        #This is not working properly, because as soon as the spell uses reflex_save_and_damage 
```

Comment:
Can you give an example?

Reply:
I did a post in the co8 forum, found it easier to attach all needed files there.

Reply:
Should be fixed now, please help test this :)